### PR TITLE
Add navbar components

### PR DESCRIPTION
**Pull request by aesha-spreadd**
1) Add navbar component

Screenshot 2023-12-29 at 10 23 37 AM

- [x] Checking the code is duplicate or not?
- [x]  Is these code has bug
- [ ]  Is code is check by shubhum sir or ankit sir or tara ma'am?
- [ ]  Are you going to merge this or not?

### CODE REVIEW

File: src/Navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+              type="cart"
+              img="https://adorn-components.netlify.app/assets/images/cart-outlined.svg"
+              count={12}
+            />
+          </NavbarIconContainer>
+
+          <NavbarLinkContainer>
+            <ul>
+              <li>
+                <a href="">Home</a>
```
Comment (tarakhant1297) [inline src/Navbar/Navbar.js]: @aeshapatel025 all links pass dynamically from stories file

File: src/Navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <StyledNavbarExample>
+          <h2>{title}</h2>
+          <NavbarContentContainer>
+            <div style={{ display: 'flex', alignItems: 'center' }}>
+              <input
+                type="text"
+                placeholder="Search"
+                style={{ marginRight: '10px' }}
+              />
+              <Button type="secondary" variant="outlined" text="Login" />
```
Comment (tarakhant1297) [inline src/Navbar/Navbar.js]: @aeshapatel025 all component in Navbar pass dynamically .

File: src/Navbar/Navbar.Styled.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import theme from '../theme';
+
+export const StyledNavbarComponent = styled.div`
+  font-family: 'Rubik', sans-serif;
+`;
+
+export const NavbarMainContainer = styled.div`
```
Comment (tarakhant1297) [inline src/Navbar/Navbar.Styled.js]: @aeshapatel025 use prefix styled to all styled component

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/Navbar/Navbar.Styled.js (hunk 3 above)
Comment (aesha-spreadd) [reply src/Navbar/Navbar.Styled.js]: changes done!

File: src/Navbar/Navbar.Styled.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  h2 {
+    color: ${theme.primary};
+    padding-left: 18px;
+    font-weight: 600;
+    font-size: 30px;
+    margin-right: 30px;
+  }
+`;
+
+export const NavbarContainer = styled.div`
```
Comment (tarakhant1297) [inline src/Navbar/Navbar.Styled.js]: @aeshapatel025 use styled prefix

File: src/Navbar/Navbar.Styled.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  align-items: center;
+  justify-content: space-between;
+`;
+
+export const StyledNavbarExample = styled.div`
+  background: rgb(246, 249, 252);
+  width: 100%;
+  display: flex;
+  align-items: center;
+  h2 {
```
Comment (tarakhant1297) [inline src/Navbar/Navbar.Styled.js:22]: @aeshapatel025 add line above this

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/Navbar/Navbar.Styled.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+  background: ${theme.textColor};
+  width: 100%;
+  display: flex;
+  align-items: center;
+
+  h2 {
+    color: ${theme.primary};
+    padding: 0 0 0 18px;
+    font-weight: 600;
+    font-size: 30px;
```
Comment (ankitadrixus08) [inline src/Navbar/Navbar.Styled.js:26]: you don't need line height here? @aeshapatel025

**Review by ankitadrixus08 — APPROVED** [review]

**Pull request merged**